sdk/textanalytics/azure-ai-textanalytics/README.md

```diff
@@ -195,6 +205,8 @@ The following section provides several code snippets covering some of the most c
 * [Recognize PII Entities](#recognize-pii-entities "Recognize pii entities")
 * [Extract Key Phrases](#extract-key-phrases "Extract key phrases")
 * [Detect Language](#detect-language "Detect language")
+* [Healthcare Analysis](#healthcare-analysis "Healthcare analysis")
+* [Batch Analysis](#analyze "Batch analysis")
```

Comment:
I think this might have to be `(#batch-analysis "Batch analysis")`

Reply:
Done!